Dockerfile

```diff
@@ -52,8 +79,4 @@ RUN { \
     echo 'SetEnvIf X-Forwarded-Proto "https" HTTPS=on'; \
   } > /etc/httpd/conf.d/remote_ssl.conf
 
-RUN rpm-file-permissions
-
 USER 1001
-
-ENTRYPOINT [ "/usr/libexec/s2i/run" ]
```

Comment:
Does this image not have an entry point atm then? It should be able to start on its own probably.

Reply:
Added entrypoint script to start apache and do some boilerplate configuration of apache/php/prefork mpm from environment variables.